src/overview/woodcutting/WoodcuttingOverview.tsx

```diff
@@ -30,17 +32,40 @@ const WoodcuttingOverview = () => {
     }
   };
 
+  const [heat] = useNumberItemObserver("heat", id);
+
   return (
-    <OverviewBox height={250} width={550} justifyContent={"space-between"}>
+    <OverviewBox height={250} width={550} justifyContent={"space-evenly"}>
       <div
         style={{
           display: "flex",
           justifyContent: "center",
           flexWrap: "wrap",
         }}
       >
+        <div
+          style={{
+            display: "flex",
+            flexDirection: "column",
+            marginTop: "5px",
+            gap: "10px",
+            width: "50px",
+            alignItems: "center",
+          }}
+        >
+          <IPimg
+            name={"heat"}
+            size={20}
+            title={Items.get_pretty_item_name("heat")}
+          />
+          <span>{heat}</span>
+        </div>
+
         {logs.map((log) => (
-          <LogDisplay log={log} key={log} />
+          <LogDisplay
+            log={log} 
+            logHeat={Cooking.LOG_HEAT_MAP[log]}
```

Comment:
This does not take into account achievements. You need to use `Cooking.getHeatPerLog(log)`